src/main/java/com/liefery/android/stop_badge/StopBadgeFactory.java

```diff
@@ -0,0 +1,37 @@
+package com.liefery.android.stop_badge;
+
+import android.annotation.SuppressLint;
+import android.graphics.*;
+import android.util.LruCache;
+
+@SuppressLint( "NewApi" )
```

Comment:
Yes, IntelliJ tells me `API 12 is needed (current API 1)` even though I set it. I think it's just IntelliJ not picking up the right value.